src/renderer/start-frame/start-frame.injectable.ts

```diff
@@ -13,19 +13,19 @@ const startFrameInjectable = getInjectable({
   // TODO: Consolidate contents of bootstrap.tsx here
   instantiate: (di) => {
     const runMany = runManyFor(di);
-    const evenBeforeFrameStarts = runMany(evenBeforeFrameStartsInjectionToken);
-    const evenBeforeMainFrameStarts = runMany(evenBeforeMainFrameStartsInjectionToken);
-    const evenBeforeClusterFrameStarts = runMany(evenBeforeClusterFrameStartsInjectionToken);
+    const beforeFrameStartsFirst = runMany(beforeFrameStartsFirstInjectionToken);
+    const beforeMainFrameStarts = runMany(beforeMainFrameStartsInjectionToken);
+    const beforeClusterFrameStarts = runMany(beforeClusterFrameStartsInjectionToken);
     const beforeFrameStarts = runMany(beforeFrameStartsInjectionToken);
     const currentlyInClusterFrame = di.inject(currentlyInClusterFrameInjectable);
 
     return async () => {
-      await evenBeforeFrameStarts();
+      await beforeFrameStartsFirst();
```

Comment:
Which frame starts? _(since below we're seeing distinguishing btw cluster/main frame)._ 
Why we should wait for?  _(just random question for the first time looking at the code)_

Reply:
Either frame, this function is here to consolidate all the initialisation steps (and the temporal dependencies between them). The reason why we split up these `Runnable`'s (which is the type we created to describe each of these steps) into separate phases (which is what each of these `runMany` calls represents) is because sometimes there is a general need to have a step complete before anything else is run.

We could have instead specified a `runAfter` for every other runnable (which is how we specify the temporal ordering dependencies) but that seemed to be too repetitive.